website/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "scripts": {
         "examples": "docusaurus-examples",
-        "start": "rimraf .docusaurus && docusaurus start",
+        "start": "rimraf .docusaurus && npm run generate && docusaurus start",
         "build": "rimraf .docusaurus && docusaurus build",
```

Comment:
i guess we should move the `generate` command here as well (and maybe to `deploy` script too, and remove it from the `docs` workflow)

Reply:
Agreed, have done that

Reply:
now we can also remove the `npm run generate` call from `docs.yml`

Reply:
Forgot to do that, woops. Committed the removal